macros/GroupData.json

```diff
@@ -1969,6 +1969,7 @@
                             "VRPose",
                             "VREyeParameters",
                             "VRFieldOfView",
+                            "VRLayer"
```

Comment:
This is a dictionary. I know we haven't really made our minds up with interfaces vs. dictionaries, but I'm rather hesitant about adding it to a list of "interfaces". What do you think? Is it an important dictionary?